crates/flux-refineck/src/checker.rs

```diff
@@ -604,7 +606,7 @@ impl<'ck, 'genv, 'tcx, M: Mode> Checker<'ck, 'genv, 'tcx, M> {
                             &resolved_args.lowered,
                         )
                         .with_src_info(terminator.source_info)?;
-
+                        let actuals = self.check_operands(infcx, env, terminator_span, args)?;
```

Comment:
Is there a reason to move this here?

Reply:
vestige of the earlier approach with loc/place etc. (reverted!)